Sources/dep/Git.swift

```diff
@@ -91,10 +91,14 @@ class Git {
         }
 
         do {
+            var environment = [String:String]()
+            if let sshAuthSockPath = getenv("SSH_AUTH_SOCK") { //for auto ssh authentication if possible
+                environment["SSH_AUTH_SOCK"] = sshAuthSockPath
```

Comment:
there is a conversation about inheriting all variables here : https://bugs.swift.org/browse/SR-588